src/metabase/driver/sync.clj

```diff
@@ -248,7 +248,7 @@
 
 
 (defn- sync-database-with-tracking! [driver database full-sync?]
-  (let [start-time (System/currentTimeMillis)
+  (let [start-time (System/nanoTime)
```

Comment:
because we have a sweet util function that nicely formats timestamps in an appropriate unit, and it works with nanoseconds. This way logging for sync now says things like "2 sec" or "1 min" or w/e instead of "2000 ms" or "60000 ms"